### DISCUSSION TITLE
Cannot figure out how to get these commands to work after installing the Alfred extension?

### DISCUSSION THREAD

**Discussion by him229** [xilopaint/alfred-pdf-tools#11] [Q&A] [answered]
I installed the Alfred extension; now what do I do to actually merge two pdf files in a folder?

**Comment by xilopaint** [comment, marked as answer]
Hey @him229, sorry for the late response. I can see on #12 that you've already figured out how to perform a file action. [A link to a post](https://www.alfredapp.com/blog/tips-and-tricks/file-actions-from-alfred-or-finder/) on Alfred blog about the topic has been added to README.
